Dockerfile.gpu-test

```diff
@@ -0,0 +1,65 @@
+FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04 AS builder
+
+ENV DEPS "build-essential git ca-certificates curl unzip wget libgomp1 patchelf coreutils tcl libjemalloc-dev"
```

Comment:
We should consider utilizing /opt/system-setup.py infrastructure for requirements installations, as we repeat them in CircleCI and native environments.